assets/js/blocks/reviews-by-category/edit.js

```diff
@@ -0,0 +1,315 @@
+									</RawHTML>
+								</Notice>
+							) }
+						</Fragment>
+					) }
+				</PanelBody>
+				<PanelBody title={ __( 'List Settings', 'woo-gutenberg-products-block' ) }>
+					<ToggleControl
+						label={ __( 'Order by', 'woo-gutenberg-products-block' ) }
+						checked={ attributes.showOrderby }
+						onChange={ () => setAttributes( { showOrderby: ! attributes.showOrderby } ) }
+					/>
+					<SelectControl
+						label={ __( 'Order Product Reviews by', 'woo-gutenberg-products-block' ) }
+						value={ attributes.orderby }
+						options={ [
+							{ label: 'Most recent', value: 'most-recent' },
+							{ label: 'Highest Rating', value: 'highest-rating' },
+							{ label: 'Lowest Rating', value: 'lowest-rating' },
+						] }
+						onChange={ ( orderby ) => setAttributes( { orderby } ) }
+					/>
+					<RangeControl
+						label={ __( 'Starting Number of Reviews', 'woo-gutenberg-products-block' ) }
+						value={ attributes.reviewsOnPageLoad }
+						onChange={ ( reviewsOnPageLoad ) => setAttributes( { reviewsOnPageLoad } ) }
+						max={ maxPerPage }
+						min={ minPerPage }
+					/>
+					<ToggleControl
+						label={ __( 'Load more', 'woo-gutenberg-products-block' ) }
+						checked={ attributes.showLoadMore }
+						onChange={ () => setAttributes( { showLoadMore: ! attributes.showLoadMore } ) }
+					/>
+					{ attributes.showLoadMore && (
+						<RangeControl
+							label={ __( 'Load More Reviews', 'woo-gutenberg-products-block' ) }
+							value={ attributes.reviewsOnLoadMore }
+							onChange={ ( reviewsOnLoadMore ) => setAttributes( { reviewsOnLoadMore } ) }
+							max={ maxPerPage }
+							min={ minPerPage }
+						/>
+					) }
+				</PanelBody>
+			</InspectorControls>
+		);
+	};
+
+	const renderEditMode = () => {
+		const onDone = () => {
+			setAttributes( { editMode: false } );
+			debouncedSpeak(
+				__(
+					'Showing Reviews by Category block preview.',
+					'woo-gutenberg-products-block'
+				)
+			);
+		};
+
+		return (
+			<Placeholder
+				icon={ <IconReviewsByCategory className="block-editor-block-icon" /> }
+				label={ __( 'Reviews by Category', 'woo-gutenberg-products-block' ) }
+				className="wc-block-reviews-by-category"
+			>
+				{ __(
+					'Show product reviews from specific categories.',
+					'woo-gutenberg-products-block'
+				) }
+				<div className="wc-block-reviews-by-category__selection">
+					<ProductCategoryControl
+						selected={ attributes.categoryIds }
+						onChange={ ( value = [] ) => {
+							const ids = value.map( ( { id } ) => id );
+							setAttributes( { categoryIds: ids } );
+						} }
+					/>
+					<Button isDefault onClick={ onDone }>
+						{ __( 'Done', 'woo-gutenberg-products-block' ) }
+					</Button>
+				</div>
+			</Placeholder>
+		);
+	};
+
+	const renderViewMode = () => {
```

Comment:
This function is also very similar between _Reviews by Product_ and _Reviews by Category_ except for the label and the class name. Still, I think we should explore moving it to another file and importing it here, so we avoid having duplicate code in two different places.